Winnie-The-Pooh.py

```diff
@@ -1,3 +1,37 @@
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, 
+# насколько легко он их придумывает, Вам стоит написать программу. 
+# Винни-Пух считает, что ритм есть, если число слогов 
+# (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, 
+# то они разделяются дефисами. Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры. 
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке 
+# и “Пам парам”, если с ритмом все не в порядке
+
+# Пример:
+# Ввод: пара-ра-рам рам-пам-папам па-ра-па-да    
+# Вывод: Парам пам-пам
+
+
+poem = input("Введите стихотворение: ")
+
+vowels = ['а', 'я', 'и', 'е', 'о', 'у', 'ю', 'э', 'е', 'ы', 'ё'] #список гласных
+words_in_poem = poem.split()#разделяем стихотворение на слова, разделенные пробелами
+print(words_in_poem)
+result = list()
+
+for word in words_in_poem: #перебираем слова в стихотворении
+    count = 0
+    for letter in word: # перебираем буквы с словах
+        if letter in vowels:
+            count += 1
```

Comment:
Смотрите, как изящно можно сделать
```suggestion
        count += letter in vowels
```
Сэкономили строку, при этом код совсем не усложнили.